resources/xsd/xml.xsd

```diff
@@ -0,0 +1,155 @@
+<?xml version='1.0'?>
+<!--
+  Copyright © 2018 World Wide Web Consortium, (Massachusetts
+  Institute of Technology, European Research Consortium for Informatics and
+  Mathematics, Keio University, Beihang). All Rights Reserved. This work is
+  distributed under the W3C® Software License [1] in the hope that it will be
+  useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+  [1] http://www.w3.org/Consortium/Legal/2002/copyright-software-20021231
+-->
```

Comment:
Many of these files you've copied have different licenses. If they are going to be copied like this (which I hope there is an alternative), then you need to update the LICENSE/NOTICE files accordingly.